lib/trailblazer/context.rb

```diff
@@ -12,23 +12,22 @@ module Trailblazer
   # :data object:
   module Context
     require "trailblazer/context/container"
-
-    module Extension
-      autoload :Aliasing, "trailblazer/context/extension/aliasing"
-    end
+    require "trailblazer/context/container/with_aliases"
 
     module_function
 
     def for_circuit(wrapped_options, mutable_options, (_, flow_options), **)
-      Context.build(wrapped_options, mutable_options, **flow_options)
+      build(wrapped_options, mutable_options, **flow_options)
     end
 
-    def build(wrapped_options, mutable_options, container_class: Context::Container, **flow_options)
-      if flow_options[Context::Extension::Aliasing::CONFIG_KEY]
-        container_class = Class.new(container_class).prepend(Context::Extension::Aliasing)
```

Comment:
Well, not a biggie but few things that I thought about this,

- "container_class" seemed kind of generic to pass in `flow_options`. It would be cool if we could have a structure like `flow_options: { context: { container_class: Context::Container, aliases: { ... } } }`

- Above check to replace "container_class" with `Context::WithAliases` when `context_alias` are present might have led to confusion.

- Which use case there could be to pass containers apart from what we've defined ?

Reply:
It's always better to avoid globals and pass options around programmatically, and since we're already in a functional flow, I think it's less error prone (always imagine you're too test this behavior and have to override a global for a certain time, only. Seen this everywhere in Rails :cry: ).

I like the structure for the options you propose!

Reply:
Cool bro. If we have that namespace for `context options`, then it's sorted!